CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing to OpenEBS istgt
+
+istgt uses the standard GitHub pull requests process to review and accept contributions.
+
+You can contribute to istgt by filling a issue at [openebs/istgt](https://github.com/openebs/istgt/issues) or submitting a pull reuqest to this repository.
+* If you want to file a issue for bug or feature request, please see [Filing a issue](#filing-a-issue)
+* If you are a first-time contributor, please see [Steps to Contribute](#steps-to-contribute) and code standard(code-standard.md).
+* If you would like to work on something more involved, please connect with the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
+## Filing a issue
+### Before filing an issue
+
+If you are unsure whether you have found a bug, please consider asking in the [Slack](https://kubernetes.slack.com/messages/openebs) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the [issue tracker](https://github.com/openebs/istgt/issues).
+
+### Filing issues
+
+When filing an issue, make sure to answer the following questions:
+
+1. What version of OpenEBS are you using?
+2. What did you expect to see?
+3. What did you see instead?
+4. How to reproduce it?
+5. Logs of cstor-istgt and cstor-volume-mgmt containerss of cstor target pod.
+
+## Steps to Contribute
+
+istgt is an Apache 2.0 Licensed project and all your commits should be signed with Developer Certificate of Origin. See [Sign your work](./code-standard.md).
+
+
+* Find an issue to work on or create a new issue. The issues are maintained at [openebs/istgt](https://github.com/openebs/istgt/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/openebs/labels/good%20first%20issue).
+* Claim your issue by commenting your intent to work on it to avoid duplication of efforts.
+* Fork the repository on GitHub.
+* Create a branch from where you want to base your work from replication branch.
+* Make your changes. If you are working on code contributions, please see [Setting up the Development Environment](#setting-up-your-development-environment).
+* Commit your changes by making sure the commit messages convey the need and notes about the commit.
+* Please make sure than your code is aligned with the standard mentioned at [code-standard](code-standard.md).
+* Verify that your changes passes `sudo bash test_istgt.sh`
```

Comment:
also need to add about code style verification.

Reply:
Updated with `./cstyle.pl <modified_file_name>`